addon/ng2/utilities/route-utils.ts

```diff
@@ -70,6 +70,9 @@ export function bootstrapItem(mainFile, imports: {[key: string]: [string, boolea
 
 export function insertImport(fileToEdit: string, symbolName: string,
                                   fileName: string, isDefault = false): Change {
+        
+  fileName = fileName.replace(/\\/, '/'); // correction in windows           
```

Comment:
1. start from blueprint/component/index.js line: 120 

 `
 const componentDir = path.relative(this.dynamicPath.appRoot, this.generatePath);
 const importPath = componentDir ? `./${componentDir}/${fileName}` : `./${fileName}`;
    if (!options['skip-import']) {
      returns.push(
        astUtils.addComponentToModule(modulePath, className, importPath)
          .then(change => change.apply()));
    }
`


2. ast-utils.js line: 291


`
export function addComponentToModule(modulePath: string, classifiedName: string,
    importPath: string): Promise<Change> {
  return _addSymbolToNgModuleMetadata(modulePath, 'declarations', classifiedName, importPath);
}
`

3.in file: ast-utils. _addSymbolToNgModuleMetadata line: 282


`const importInsert: Change = insertImport(ngModulePath, symbolName, importPath);`


the importPath is carried through from the beginning.(blueprint/component and others). 
It seems componentDir is wrong. path.relative(windows environment) will use '\' to return relative path. 


By the way. in route-utils.ts line:205. I used the same trick to fix #1719